### ISSUE TITLE
Use on-device OsmAnd maps instead of downloading them ?

### ISSUE THREAD

**Issue by breversa** [mozilla/MozStumbler#1465]
Hi there,

For people who use both OsmAnd and MozStumbler, would it be possible that MozStumbler uses OsmAnd maps for display, instead of downloading them again (hence duplicating) ? I'm not thinking of displaying cells IN OsmAnd, just using OsmAnd map data in MozStumbler.

Advantages I see :
- No additionnal use of data, be it wifi or phone network
- High-detail maps all the time
- Some device space saved thanks to deduplication

However, I don't know if an app can use data from another app. And, of course, for people who don't use OsmAnd (or not the same regions), that point is moot.

**Comment by crankycoder** [comment]
We can't do that.  Sharing tiles has numerous problems with respect to caching.  OsmAnd assumes that all applications have exactly the same use-case for the tiles - and we don't fit in there.  We used to use the same tile directory but it just lead to problems with incorrect tiles, or mismatched tiles when the stumbler downloaded tiles collided with OsmAnd tiles.

I'm closing this as this is actually a feature we used to have, but have made an explicit choice to remove.
